content/get-started/index.md

```diff
@@ -0,0 +1,78 @@
+
+| Lesson |  Slides | Hands-on | Recording | Interactive Tour |  
+|---|---|---|---|---|
+| A short introduction to Galaxy | [<i class="fab fa-slideshare" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/introduction/tutorials/galaxy-intro-short/slides.html#1){:target="_blank"}   | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/introduction/tutorials/galaxy-intro-short/tutorial.html){:target="_blank"} |  |   |   |
+| Galaxy 101 |   | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/introduction/tutorials/galaxy-intro-101-everyone/tutorial.html){:target="_blank"}   | [<i class="fas fa-video" aria-hidden="true"></i>](https://gallantries.github.io/video-library/videos/introduction/galaxy-intro-101/tutorial/){:target="_blank"}    |   |
+| Your first Galaxy Analysis | [<i class="fab fa-slideshare" aria-hidden="true"></i>](bit.ly/smorg-2021-intro){:target="_blank"}  |   |[<i class="fas fa-video" aria-hidden="true"></i>](https://youtu.be/1UPCrQxGwcU){:target="_blank"} |   |
+| Galaxy Tour  |   |   |   | [<i class="fas fa-mouse" aria-hidden="true"></i>](https://usegalaxy.org/tours/core.galaxy_ui){:target="_blank"} |
+| Galaxy Histories |   | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/galaxy-interface/tutorials/history/tutorial.html){:target="_blank"}  |   | [<i class="fas fa-mouse" aria-hidden="true"></i>](https://usegalaxy.org/tours/core.history){:target="_blank"} |
+| How to reproduce published Galaxy Analysis |  | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/introduction/tutorials/galaxy-reproduce/tutorial.html){:target="_blank"} |   |   |
+|Introduction to the GTN |[<i class="fab fa-slideshare" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/contributing/slides/introduction.html#1
+){:target="_blank"} |   | [<i class="fas fa-video" aria-hidden="true"></i>](https://www.youtube.com/watch?v=lDqWxzWNk1k){:target="_blank"} |   |
+| How to contribute using Gitpod |   | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/gitpod/tutorial.html){:target="_blank"}  | [<i class="fas fa-video" aria-hidden="true"></i>](https://youtu.be/_-wnsPttBCo){:target="_blank"}  |   |
+|How to create a new tutorial in the GTN   |   |[<i class="fa fa-laptop" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial/tutorial.html){:target="_blank"}   |[<i class="fas fa-video" aria-hidden="true"></i>](https://www.youtube.com/watch?v=Vw9h5kvRH2k){:target="_blank"}  |   |   |
+
```

Comment:
I would put a link here about the Galaxy Community. And before setting up your community, link to regional communities, scientific communities. and only if you don't find anything that suit you you could consider setting up your community.

And I would probably change the title to something a bit mroe "soft". "What if you can't find yourself in these lists of regional and/or scientific communities?)

### Set up your community

Reply:
Okay,agreed!  Two separate titles it is then